Clinical-Genomics/scout
# Bootstrap4 ready-sh

This PR changes the layout of Scout and introduces nice graphs in the dashboard pages

**How to test**:
1. install the branch on clinical-db so that the changes will be in scout-stage:
```
bash update-scout-stage.sh bootstrap4_ready
 ```
1. Extensive tests should be performed by checking all web pages available.
1. Colors and objects are not fixed and can (and should) be changed. **I'm just suggesting a style** but we can discuss about it together to improve the portal.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] tests executed by dNil, MMA, CR etc
